rtd_requirements.txt

```diff
@@ -1,2 +1,3 @@
 django>=1.9,<1.10
 six
+beautifulsoup4
```

Comment:
The `` rtd_requirements.txt `` consist requirements for build docs on http://django-mailbox.readthedocs.io/. If you like add new requirements to install use ``install_requires`` in `/setup.py` instead.